static/elements/chromedash-upcoming.js

```diff
@@ -0,0 +1,95 @@
+import {LitElement, html} from 'lit-element';
+import '@polymer/iron-icon';
+import style from '../css/elements/chromedash-upcoming.css';
+
+const TEMPLATE_CONTENT = {
+  stable: {
+    channelLabel: 'Stable',
+    h1Class: '',
+    downloadUrl: 'https://www.google.com/chrome/browser/desktop/index.html',
+    downloadTitle: 'Download Chrome stable',
+    dateText: 'was',
+    featureHeader: 'Features in this release',
+  },
+  beta: {
+    channelLabel: 'Next up',
+    h1Class: 'chrome_version--beta',
+    downloadUrl: 'https://www.google.com/chrome/browser/beta.html',
+    downloadTitle: 'Download Chrome Beta',
+    dateText: 'between',
+    featureHeader: 'Features planned in this release',
+  },
+  dev: {
+    channelLabel: 'Dev',
+    h1Class: 'chrome_version--dev',
+    downloadUrl: 'https://www.google.com/chrome/browser/canary.html',
+    downloadTitle: 'Download Chrome Canary',
+    dateText: 'coming',
+    featureHeader: 'Features planned in this release',
+  },
+};
+const REMOVED_STATUS = ['Removed'];
+const DEPRECATED_STATUS = ['Deprecated', 'No longer pursuing'];
+const SHOW_DATES = true;
+
+class ChromedashUpcoming extends LitElement {
+  static styles = style;
+
+  static get properties() {
+    return {
+      // Assigned in upcoming-page.js,
+      channels: {attribute: false},
+      showBlink: {attribute: false},
+      signedIn: {type: Boolean},
+      loginUrl: {type: String},
+      starredFeatures: {type: Object},
```

Comment:
It would be good to indicate that this is going to hold a Set.  Maybe you could just do that in a comment.

Reply:
Okay. Done. See [this](https://github.com/GoogleChrome/chromium-dashboard/pull/1432/commits/6b0ee35dea11411c0a7eb5db5c9420a0bc7cae8d) commit.